### DISCUSSION TITLE
Welcome to blog-post-workflow Discussions!

### DISCUSSION THREAD

**Discussion by gautamkrishnar** [gautamkrishnar/blog-post-workflow#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shriekdj** [comment]
IDK What To tell Mr. Gautam, But thanks for Creating this workflow it helped me a lot 🙏

**Reply by gautamkrishnar** [reply]
Thanks a lot. Glad you liked it 😄
